packages/server/app.js

```diff
@@ -20,44 +20,25 @@ const { ApolloServer, ForbiddenError } = require('apollo-server-express')
 
 const { buildContext } = require('./modules/shared')
 const knex = require('./db/knex')
+const { buildErrorFormatter } = require('@/modules/core/graph/setup')
+const { isDevEnv, isTestEnv } = require('@/modules/core/helpers/envHelper')
 
 let graphqlServer
 
 /**
- * Initialises the express application together with the graphql server middleware.
- * @return {[type]} an express application and the graphql server
+ * Create Apollo Server instance
+ * @param {Partial<import('apollo-server-express').ApolloServerExpressConfig>} optionOverrides Optionally override ctor options
+ * @returns {import('apollo-server-express').ApolloServer}
  */
-exports.init = async () => {
-  const app = express()
-
-  Logging(app)
-  MatStartup()
+exports.buildApolloServer = (optionOverrides) => {
```

Comment:
Moved out ApolloServer creation to a function so that tests can create their own instances